src/index.html

```diff
@@ -52,8 +52,213 @@ <h1>Dnn HTML custom elements</h1>
+                  ],
+                  "locked": false,
+                  "default": true
+              },
+              {
+                  "roleId": 1,
+                  "roleName": "Registered Users",
+                  "permissions": [],
+                  "locked": false,
+                  "default": true
+              }
+          ],
+          "userPermissions": [
+              {
+                  "userId": 2,
+                  "displayName": "Daniel Valadas",
+                  "permissions": [
+                      {
+                          "permissionId": 8,
+                          "permissionName": "Browse Folder",
+                          "permissionKey": null,
+                          "permissionCode": null,
+                          "fullControl": false,
+                          "view": false,
+                          "allowAccess": true
+                      },
+                      {
+                          "permissionId": 5,
+                          "permissionName": "View Folder",
+                          "permissionKey": null,
+                          "permissionCode": null,
+                          "fullControl": false,
+                          "view": false,
+                          "allowAccess": true
+                      },
+                      {
+                          "permissionId": 6,
+                          "permissionName": "Write to Folder",
+                          "permissionKey": null,
+                          "permissionCode": null,
+                          "fullControl": false,
+                          "view": false,
+                          "allowAccess": true
+                      }
+                  ]
+              }
+          ]
+        };
+        const roleGroups = [
+            {
+                "id": 0,
+                "name": "Test group",
+                "rolesCount": 0,
+                "description": "This is a a test role group"
+            },
+            {
+              "id": 1,
+              "name": "Another test group",
+              "rolesCount": 0,
+              "description": "Another test role."
+            }
+        ];
+        var roles = [
+          {
+            IsSystemRole: true,
+            RoleId: 0,
+            RoleGroupId: -1,
+            RoleName: "Administrators",
+          },
+          {
+            IsSystemRole: true,
+            RoleId: 1,
+            RoleGroupId: -1,
+            RoleName: "Registered Users",
+          },
+          {
+            IsSystemRole: true,
+            RoleId: 5,
+            RoleGroupId: -1,
+            RoleName: "Translator (fr-FR)",
+          },
+          {
+            IsSystemRole: true,
+            RoleId: -3,
+            RoleGroupId: -1,
+            RoleName: "Unauthenticated Users",
+          },
+          {
+            IsSystemRole: false,
+            RoleId: 10,
+            RoleGroupId: 1,
+            RoleName: "Test Role",
```

Comment:
```suggestion
            roleName: "Test Role",
```

Reply:
Same for this, I wrote it to match the existing APIs, I can change it but it would also mean changing the upstream APIs in Dnn or the consumer having to map the two objects only for a casing issue. In the resource-manager it's not that bad, but some APIs come from the DNN InternalServices which are global and we can't afford to change the casing there as it would break all other modules consuming that and would be a breaking change for DNN.

Reply:
Yikes - another waste of time.  😂👍🏼 Sorry about that.